package.json

```diff
@@ -2909,5 +3453,10 @@
 	"dependencies": {
 		"jsonc-parser": "^2.3.0",
 		"lodash": "^4.17.19"
+	},
+	"__metadata": {
+		"id": "1c81ab96-0424-43c4-b356-fe408a1bd1cf",
+		"publisherDisplayName": "VSpaceCode",
+		"publisherId": "60415ab6-4581-4e73-a7e0-6fc6b3369f12"
 	}
```

Comment:
What's this?

Reply:
I saw a previous comment from Steven that suggested to remove it

Reply:
Good catch. I didn't see it on the second time.

Reply:
done.  thank you

Reply:
did you forget to push, doesn't seems to have update?

Reply:
Oh, that change slipped in again when I removed the duplicate key.  Sorry for repeated mistake